lancache-litmus/overlay/var/www/litmus/index.html

```diff
@@ -1,208 +1,697 @@
+			padding: 1.75rem;
+			box-shadow: var(--shadow);
+			border: 2px solid rgba(255,255,255,0.1);
+			position: relative;
+			display: flex;
+			flex-direction: column;
+			transition: transform .3s, box-shadow .3s;
 		}
-	</style>
+
+        .cdn-card:hover {
+            transform: translateY(-4px);
+            box-shadow: var(--shadow-hover);
+        }
+
+        .cdn-card.status-success   { border-top: 6px solid var(--success-color); }
+		.cdn-card.status-warning   { border-top: 6px solid var(--warning-color); }
+		.cdn-card.status-danger    { border-top: 6px solid var(--danger-color); }
+		.cdn-card.status-loading   { border-top: 6px solid rgba(100,100,100,.5); }
+
+        .cdn-header {
+            display: flex;
+            align-items: center;
+            margin-bottom: 1rem;
+            flex-shrink: 0;
+        }
+
+        .cdn-logo {
+            width: 40px;
+            height: 40px;
+            border-radius: 8px;
+            margin-right: 1rem;
+            background: rgba(233, 29, 18, 0.1);
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            font-weight: 700;
+            color: var(--text-color);
+            flex-shrink: 0;
+            border: 1px solid var(--primary-color);
+        }
+
+        .cdn-title {
+            font-size: 1.3rem;
+            font-weight: 700;
+            color: var(--text-color);
+            text-transform: capitalize;
+            line-height: 1.2;
+        }
+
+        .cdn-description {
+            color: var(--text-color);
+            font-size: 0.95rem;
+            margin-bottom: 1.5rem;
+            flex-grow: 1;
+            line-height: 1.5;
+            opacity: 0.8;
+        }
+
+        .cdn-footer {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            margin-top: auto;
+            padding-top: 1rem;
+            border-top: 1px solid var(--primary-color);
+        }
+
+        .status-indicator {
+            display: flex;
+            align-items: center;
+            font-weight: 600;
+            font-size: 1rem;
+        }
+
+        .status-dot {
+            width: 12px;
+            height: 12px;
+            border-radius: 50%;
+            margin-right: 0.75rem;
+            position: relative;
+        }
+
+        .status-dot.success {
+            background: var(--success-color);
+        }
+
+        .status-dot.warning {
+            background: var(--warning-color);
+        }
+
+        .status-dot.danger {
+            background: var(--danger-color);
+        }
+
+        .status-dot.loading {
+            background: var(--gray);
+            animation: pulse 2s infinite;
+        }
+
+        .status-dot.success::after {
+            content: '✓';
+            position: absolute;
+            top: -1px;
+            left: 2px;
+            color: white;
+            font-size: 10px;
+            font-weight: bold;
+        }
```

Comment:
```suggestion
        .status-dot.success::after {
            content: '✓';
            display: block;
            height: 100%;
            color: white;
            font-size: calc(var(--status-dot-size) / 2);
            font-weight: bold;
            text-align: center;
            line-height: var(--status-dot-size);
        }
```
Ideally the `font-size` would scale properly with the `--status-dot-size`, but this gets a little tricky. Might be simpler to make that a variable next to` --status-dot-size`.